src/components/book.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Book({
+  name, author, category, progress,
+}) {
+  return (
+    <div className="d-flex book px-4 py-4 rounded mb-5">
+      <div style={{ width: '60%' }} className="d-flex justify-content-between">
+        <div className="">
+          <h5>{category}</h5>
+          <h4>{name}</h4>
+          <h6>{author}</h6>
+          <div className="d-flex gap-3">
+            <p>Comments</p>
+            <p>|</p>
+            <p>Remove</p>
```

Comment:
This should be a clickable element either a link or button in order for it to be possible to perform deletes with it.

Reply:
done!